src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -462,6 +462,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     -->
 
   <Target Name="_ComputeTransitiveProjectReferences"
+          Condition="'$(DisableImplicitTransitiveReferences)' != 'true'"
```

Comment:
I don't believe this is needed from a correctness perspective. But it seemed to avoid unnecessary computation.